examples/todo/src/controllers/todo.controller.ts

```diff
@@ -70,6 +71,22 @@ export class TodoController {
     return this.todoRepo.findById(id);
   }
 
+  @deprecated()
```

Comment:
@mschnee , same question as in #4416. Not sure why the todo controller in our examples needs to change.
I think the code snippet examples you provide in your docs are sufficient.

Reply:
I will make this modification, however I am waiting for the full scope of changes to #4416 to be accepted first, so that I can apply it all here as well.